Tests/Functional/Sources/ApplicationStateHandlingTest.m

```diff
@@ -78,6 +78,30 @@ - (void)testOpenSettingsApplicationAndReturning_flaky {
       assertWithMatcher:grey_sufficientlyVisible()];
 }
 
+#pragma mark - Test app relaunch
+// Test lauching app post termination without causing issues in loading
```

Comment:
use /** */ style please.

Reply:
I had noticed that comments in line# 70 were using `//` style and I just followed that. Anyways, updated as per suggestion.

Reply:
Yeah, sorry about that. We're trying to update them to /** */ gradually.